### PR TITLE
Allow different dimensional scales for entr/detr

### PR DESCRIPTION
**Pull request by costachris**
Allow for different dimensional scales in entrainment/detrainment closure.

### CODE REVIEW

File: src/closures/entr_detr.jl [hunk 1]
```diff
@@ -76,45 +76,52 @@ function entrainment_inv_length_scale(
...
-        εδ_vars.b_up,
-        εδ_vars.b_en,
-        εδ_vars.w_up,
-        εδ_vars.w_en,
-        εδ_vars.tke_en,
-        εδ_vars.zc_i,
-        entr_dim_scale,
-    )
+    entr_dim_scale = entrainment_inv_length_scale(param_set, εδ_vars, entr_dim_scale)
+    detr_dim_scale = entrainment_inv_length_scale(param_set, εδ_vars, detr_dim_scale)
```
Comment (ilopezgp) [inline src/closures/entr_detr.jl:109]: I think here you are modifying the inputs, which are also called `entr_dim_scale`, ... Could you call these `ε_dim_scale` and `δ_dim_scale`?

**Review by ilopezgp — APPROVED** [review]
LGTM

**Review by charleskawczynski — APPROVED** [review]
LGTM, is this exercised?

**Comment by costachris** [comment]
> LGTM, is this exercised?
It will be used in calibration. I did some simple tests locally to make sure the dispatch was at least working.

**Review by yairchn — APPROVED** [review]
LGTM

**Pull request merged**